### ISSUE TITLE
Option for no focal unit B

### ISSUE THREAD

**Issue by cmahony** [joeyklee/bc-climate-explorer#21]
I think a lot of users will want to just view a single focal unit. is there a way to default to showing a single focal unit, and then choose to turn on the comparison?

**Comment by joeyklee** [comment]
Yes, sure this is entirely possible. We can just leave the initial state without inputs for focal unit B. 
Noted! I'll be working on it.
